src/mediapipe_internal/pythonnoderesource.cpp

```diff
@@ -50,23 +51,22 @@ Status PythonNodeResource::createPythonNodeResource(std::shared_ptr<PythonNodeRe
     std::string parentPath = fsHandlerPath.parent_path();
     std::string filename = fsHandlerPath.filename();
 
+    py::gil_scoped_acquire acquire;
```

Comment:
Does it have to be moved here ?

Reply:
I moved it because if we acquire gil inside try {...} then we don't have it in catch{...} blocks